__init__.py

```diff
@@ -44,7 +44,8 @@
             print("The Retriever lacks permission to access the ~/.retriever/ directory.")
             raise
 SCRIPT_SEARCH_PATHS = [
-    os.path.join(HOME_DIR, 'scripts/'),
+    'scripts',
+    os.path.join(HOME_DIR, 'scripts/')
 ]
```

Comment:
The original concept of search `.` was to facilitate users including their own scripts that they hadn''t contributed to the retriever. That said, I'm not aware of folks using this, but that doesn't mean that they aren't. Is there a reason that you see for removing it?